[2020-04-17 11:22:24,811] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: atvt_training.dummy_task 2020-04-17T02:21:00+00:00 [queued]>
[2020-04-17 11:22:24,817] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: atvt_training.dummy_task 2020-04-17T02:21:00+00:00 [queued]>
[2020-04-17 11:22:24,817] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-04-17 11:22:24,817] {taskinstance.py:867} INFO - Starting attempt 1 of 2
[2020-04-17 11:22:24,817] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-04-17 11:22:24,898] {taskinstance.py:887} INFO - Executing <Task(DummyOperator): dummy_task> on 2020-04-17T02:21:00+00:00
[2020-04-17 11:22:24,909] {standard_task_runner.py:53} INFO - Started process 113181 to run task
[2020-04-17 11:22:25,048] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: atvt_training.dummy_task 2020-04-17T02:21:00+00:00 [running]> bi02.com
[2020-04-17 11:22:25,058] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=atvt_training, task_id=dummy_task, execution_date=20200417T022100, start_date=20200417T022224, end_date=20200417T022225
[2020-04-17 11:22:34,722] {logging_mixin.py:112} INFO - [2020-04-17 11:22:34,722] {local_task_job.py:103} INFO - Task exited with return code 0
